src/dispatchOnMount.js

```diff
@@ -0,0 +1,23 @@
+import { Component } from 'react';
+import { Subscription } from 'rxjs/Subscription';
+
+const $$reduxObservableSubscription = '@@reduxObservableSubscription';
+
+export function dispatchOnMount(...toDispatch) {
+  return (ComposedComponent) =>
+    class DispatchOnMountComponent extends Component {
+      componentDidMount() {
+        this[$$reduxObservableSubscription] = new Subscription();
+        toDispatch.map(a => this.context.store.dispatch(a))
```

Comment:
How does this work without
```js
static contextTypes = {
  store: PropTypes.shape({
    dispatch: PropTypes.func.isRequired,
  })
};
```
I thought React required you to provide `contextTypes` to opt-in to receiving a context, otherwise it's not provided?? Am I drunk?

The docs even say so https://facebook.github.io/react/docs/context.html#passing-info-automatically-through-a-tree
> If contextTypes is not defined, then this.context will be an empty object.